code/game/turfs/flooring/flooring.dm

```diff
@@ -151,6 +151,18 @@ var/list/flooring_types
 	build_type = /obj/item/stack/tile/grass/sif
 	has_base_range = 1
 	
+	/turf/simulated/floor/outdoors/attackby(obj/item/C, mob/user)//CHOMPedit begin
```

Comment:
More appropriate to add this chunk to the original outdoors/attackby override in outdoors.dm, doesn't make sense to split proc overrides across files when it isn't specific to a child. You've also put this inside of /decl/flooring/grass/sif, which... I have no idea what that will do.

Splitting this way may result in your proc overriding (or being overriden by) the code chunk below. Particularly bad if it's overriding the below code, since that calls the parent proc + parent return values.

![image](https://user-images.githubusercontent.com/41974248/224204795-1eade6f4-67d2-473a-a79e-7bfe8b56739d.png)